### PR TITLE
Project level visualizations

### PR DESCRIPTION
**Pull request by Jkrzy**
## Description

For #1217, Expanding on the good works of NeilB to add project specific charting.

On the project details page, we now render area charts describing hours tocked by user and reporting period. This includes some tweaks to the details page to accommodate the new content.
A table containing the plotted data is also provided along with an export button. 

The table may extend horizontally quite a ways as the number of folks who've tocked to a specific project grows.

### CODE REVIEW

File: tock/tock/templates/employees/user_list.html [hunk 1]
```diff
@@ -14,7 +14,6 @@ <h1>Tock Users</h1>
       <th>First Name</th>
       <th>Last Name</th>
       <th>Start Date</th>
-      <th>End Date</th>
```
Comment (Jkrzy) [inline tock/tock/templates/employees/user_list.html:17]: Removing an unused column

File: tock/utilization/views.py [hunk 2]
```diff
@@ -58,29 +53,34 @@ def get_queryset(self):
...
+        context.update({'org_totals': org_billing_context()})
         return context
 
 
-class UtilizationAnalyticsView(LoginRequiredMixin, TemplateView):
-    template_name = "utilization/utilization_analytics.html"
-
-    def get_context_data(self, **kwargs):
-        context = super(UtilizationAnalyticsView, self).get_context_data(**kwargs)
+class FilteredAnalyticsView(LoginRequiredMixin):
```
Comment (neilmb) [inline tock/utilization/views.py:60]: I don't see where this refactored class was used elsewhere. Was there a different version where this new super-class was useful? I don't have a problem with factoring this functionality out, but it doesn't seem related to the rest of this PR.

File: tock/tock/templates/projects/project_detail.html [hunk 3]
```diff
@@ -1,99 +1,100 @@
...
-  <li> <b> Alerts: </b>
-{% if object.alerts.exists %}
-    {% for alert in object.alerts.all %}
-        {% if alert.destination_url %}
-          <a href="{{ alert.destination_url }}" target="_blank"><span class="{{ alert.full_style }}">{{ alert.full_alert_text }}</span></a>
-        {% else %}
+<div class="grid-container">
+  <div class="grid-row">
+    <div class="tablet:grid-col">
+      <ul>
```
Comment (neilmb) [inline tock/tock/templates/projects/project_detail.html:19]: Semantically I wonder if this is a worthy use of a `<dl>`? Instead of styling field names manually with `<b>`, could use `<dt>` and `<dd>`. But maybe it's not worth the effort of styling that infrequently used element?

**Review by neilmb — APPROVED** [review]
Glad that the original analytics stuff is proving to be reusable. LGTM apart from quibbles.

File: tock/utilization/views.py (hunk 2 above)
Comment (Jkrzy) [reply tock/utilization/views.py:60]: Thanks @neilmb, that's definitely unrelated to the functionality and a remnant I didn't completely walk back.

My[ original approach](https://github.com/18F/tock/pull/1221/commits/ca2d2cfc8703789d7d3f8fa5fe8c4a847787b6da#diff-cdf622b15efe20a10cd3c1e46afc2d717720d093cbc6b11dda8028c24e0a3c24R165) had the project charts on another `utilization/` page and the associated view alongside this.  When I switched gears to handling it in the projects app I left `FilteredAnalyticsView`.

We'll leave it in but I should have called it out specifically or otherwise separated from the rest of these changes.

File: tock/tock/templates/projects/project_detail.html (hunk 3 above)
Comment (Jkrzy) [reply tock/tock/templates/projects/project_detail.html:19]: Likely and I'm definitely guilty here of not reworking the vast majority of the original html from `project_detail.html` and only made the changes necessary to get the charts rendering well-enough.

I'll file another issue to look at cleaning this up a bit, thanks @neilmb!
